src/tactic/squeeze.lean

```diff
@@ -66,10 +77,13 @@ def squeeze_loc_attr_carrier := ()
 run_cmd squeeze_loc_attr.set ``squeeze_loc_attr_carrier none tt
 
 /-- Format a list of arguments for use with `simp` and friends. This omits the
-list entirely if it is empty. -/
+list entirely if it is empty.
+
+Patch: `pp` was changed to `to_string` because it was getting rid of prefixes
+that would be necessary for some disambiguations. -/
 meta def render_simp_arg_list : list simp_arg_type → tactic format
```

Comment:
By my reading, this no longer needs the `tactic` monad, right?

Reply:
yes; I'm not sure how to remove the `functor.map` yet though! (yet to read about monadic programming fully)

Reply:
Arthur sorted this!